proxy/limits/limits.go

```diff
@@ -27,31 +27,31 @@ import (
 
 var (
 	// For overriding in unittests.
-	getRlimitFunc = syscall.Getrlimit
-	setRlimitFunc = syscall.Setrlimit
+	syscallGetrlimit = syscall.Getrlimit
+	syscallSetrlimit = syscall.Setrlimit
 )
 
 // SetupFDLimits ensures that the process running the Cloud SQL proxy can have
 // at least wantFDs number of open file descriptors. It returns an error if it
 // cannot ensure the same.
 func SetupFDLimits(wantFDs uint64) error {
 	curr := &syscall.Rlimit{}
-	if err := getRlimitFunc(syscall.RLIMIT_NOFILE, curr); err != nil {
+	if err := syscallGetrlimit(syscall.RLIMIT_NOFILE, curr); err != nil {
 		return fmt.Errorf("failed to read rlimit for max file descriptors: %v", err)
 	}
 
 	if curr.Max < wantFDs {
 		return fmt.Errorf("max FDs rlimit set to %d, cannot support 4K connections. If you are the system administrator, consider increasing this limit.", curr.Max)
 	}
 
-	if curr.Cur > wantFDs {
+	if curr.Cur >= wantFDs {
 		logging.Infof("current FDs rlimit set to %d, wanted limit is %d. Nothing to do here.", curr.Cur, wantFDs)
```

Comment:
I think logging.Verbosef is more appropriate, seems like "doing nothing" is a verbose message.

Reply:
Done.